airflow-core/src/airflow/ui/src/utils/hitl.ts

```diff
@@ -125,10 +125,10 @@ export const getHITLParamsDict = (
           minLength: undefined,
           section: undefined,
           title: key,
-          type: valueType,
+          type: typeof value === "number" ? "number" : "string",
           values_display: undefined,
         },
-        value: preloadedHITLParams[key] ?? value,
+        value: paramData?.value ?? paramData ?? value,
```

Comment:
```suggestion
        value: paramData?.value ?? value,
```

I think we could remove the paramData here

Reply:
updated. Thanks!